packages/smooth_app/lib/pages/preferences/user_preferences_account.dart

```diff
@@ -361,12 +366,38 @@ class _UserPreferencesPageState extends State<UserPreferencesSection> {
     return Column(children: result);
   }
 
+  Future<int?> _getMyCount(
+    final UserProductSearchType type,
+  ) async {
+    final UserProductSearchQueryConfiguration configuration =
+        UserProductSearchQueryConfiguration(
+      type: type,
+      userId: OpenFoodAPIConfiguration.globalUser!.userId,
+      pageSize: 1,
+      language: ProductQuery.getLanguage(),
+      fields: [],
```

Comment:
```suggestion
     fields: <ProductField>[],
```

Reply:
committing this will fix the analyzer test not passing

Reply:
Done👍🏻

Reply:
@AshAman999 How come you can make suggestions and I can't anymore?

Reply:
> Done👍🏻

@prathamsoni11 I'm afraid not, or you've forgotten to push. Alternate solution: click on "commit suggestion".

Reply:
>How come you can make suggestions and I can't anymore?

There is something definitely going in with your GitHub lately 😅